assets/src/legacy/attributeTable.js

```diff
@@ -335,8 +343,14 @@ var lizAttributeTable = function() {
                         allColumnsKeyValues[columnName] = keyValue;
 
                     }
-                    lizMap.config.layers[layerName]['columns'] = responses[1].columns;
-                    buildLayerAttributeDatatable(layerName, tableSelector, responses[0].features, responses[1].aliases, responses[1].types, allColumnsKeyValues, callBack);
+                    layerConfig['featureCrs'] = 'EPSG:4326';
+                    if (namedRequests?.['describeFeatureType']) {
+                        const describeFeatureTypeResponse = responses[namedRequests['describeFeatureType']];
+                        layerConfig['aliases'] = describeFeatureTypeResponse.aliases;
+                        layerConfig['types'] = describeFeatureTypeResponse.types;
+                        layerConfig['columns'] = describeFeatureTypeResponse.columns;
+                    }
```

Comment:
Shouldn't we test that `describeFeatureTypeResponse` contains `aliases`, `types` and `columns`

Reply:
This request with json output format has been created for that - it is a dedicated lizmap request.

Reply:
I have added a test, to check each properties from the request.